models/offer.js

```diff
@@ -0,0 +1,65 @@
+const { Schema, model } = require('mongoose')
+const {
+  enums: { SPOKEN_LANG_ENUM, SUBJECT_LEVEL_ENUM }
+} = require('~/consts/validation')
+const {
+  roles: { STUDENT, TUTOR }
+} = require('~/consts/auth')
+
+const offerSchema = new Schema(
+  {
+    price: {
+      type: Number,
+      required: [true, 'This field cannot be empty.'],
+      min: [1, 'Price must be positive number']
+    },
+    proficiencyLevel: {
+      type: String,
+      enum: SUBJECT_LEVEL_ENUM,
+      required: [true, 'This field cannot be empty.']
+    },
+    description: {
+      type: String,
+      minlength: [1, 'Last Name cannot be shorter than 1 symbol.'],
```

Comment:
nice ctrl c ctrl v :D 
not last name, but description

Reply:
fixed